content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md

```diff
@@ -0,0 +1,13 @@
+# Planning and executing a migration strategy
+
+## Key points
+
+* It is a journey: do not jump into the water to try how deep it is [What Is Your Cloud Migration Strategy?](https://www.cloudstrategymag.com/articles/86667-what-is-your-cloud-migration-strategy)
```

Comment:
It is a journey: do not jump head first into the water to test how deep it is```<a href="https://www.cloudstrategymag.com/articles/86667-what-is-your-cloud-migration-strategy" target="_blank">What Is Your Cloud Migration Strategy?</a>```

Reply:
Links are opening on a new windows.